subjects/gaming/stealth-boom/audit/README.md

```diff
@@ -24,46 +14,70 @@
+###### Does the map contain pickable ammunition and health placed around?
 
-###### Whenever a drone turns to chasing mode, do all the guards in the area get alerted and switch to chasing mode as well?
+##### Player Mechanics
 
-###### Does the drone come back to patrol mode when the player is out of the drone sight?
+###### Does the player have all the minimal animation required (walking, running, melee attacking, shooting, ducking)?
 
-###### Is the sight radius of the drones smaller than the guards?
+###### Is there a sound for the guns shooting?
 
-###### Are cameras attached to walls?
+###### Is there a widget displaying the amount of bullets the player has?
 
-###### Do cameras have similar light sign as the drones (red for alert mode and blue for patrol)?
+###### Is the player unable to shoot when he has no bullets?
 
-###### As the drones, do the cameras alert guards on the area, switching them to chasing mode?
+###### Are there bullets impacts present when shooting at a wall?
 
-###### Do Guards, Drones and Cameras play an alert sound when a player gets detected?
+###### Does the player health decreases when he gets damaged?
 
-###### Do some cameras lock access to certain areas of the map, when they detect a player?
+###### When the player is hit, are there any blood particles?
 
-###### Can the camera close some part of the map (thru closed doors, open traps and tries to kill the player etc…) to the player when the player is being detected?
+###### Does the player have a health bar?
 
-###### Are Behavior Trees used to implement the AI of the Guards and Drones?
+###### When the player loses all his health (dies), does he get to choose whether to quit the game, go back to the main menu, or to start over?
 
 ###### Does the player have a goal?
 
 ###### When the goal of the player is successfully completed, does a pop up appear saying that the mission is completed?
 
-###### Does the player have a health bar?
+##### Enemies
 
-###### Does the player health decreases when he gets shot by the guards?
+###### Does the game contain at least 2 enemy types (ranged and melee)
 
-###### When the player loses all his health (dies), does he get to choose whether to quit the game, go back to the main menu, or to start over?
+###### Are Behavior Trees used to implement the AI of the enemies?
 
-###### If the player starts over does he spawn back at the starting point?
+###### Do the enemies wander around the map?
+
+###### When a player enters the field of view of an enemy, does it switch to chasing mode
+
+###### Do melee enemies approach the player to perform melee attacks?
+
+###### Do ranged enemies take cover and shoot at the player?
+
+###### do the enemies go back to patrol mode when the player is hidden from all field of views for a set amount of time?
+
+###### do the enemies have a sound effect and visual effect when entering and exiting chase mode?
+
+###### are enemies in chase mode alerting nearby enemies ?
```

Comment:
Lowercase at start of the question, must be upper

Reply:
changed